package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "restaurant-page",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
```

Comment:
[OPTIONAL]

Consider adding `webpack-dev-server` as a dev dependency for this project and add a start script associated.